dev-docs/bidders/publir.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bidder
+title: Publir
+description: Prebid Publir Bidder Adapter
+multiformat_supported: will-bid-on-any
+pbjs: true
+biddercode: publir
+media_types: banner
+schain_supported: true
+gdpr_supported: true
```

Comment:
```suggestion
tcfeu_supported: false
gvl_id: none
```

`gdpr_supported` is deprecated. If you don't have a `gvl_id`, then `tcfeuo_supported` must be false, because without a GVL ID, consent can't be enforced from a technical point of view.

The docs don't provide any info if you are legally supporting GDPR, just if you technically support TCF